Formula/hab.rb

```diff
@@ -0,0 +1,14 @@
+class Hab < Formula
+  desc "Automation that Travels with your App"
+  homepage "https://www.habitat.sh"
+  url "https://api.bintray.com/content/habitat/stable/darwin/x86_64/hab-0.6.0-20160613151520-x86_64-darwin.zip?bt_package=hab-x86_64-darwin"
```

Comment:
We'll want to build this from source. Homebrew (the formula side) doesn't accept upstream binaries unless they're something like a `.jar`. The [Cask](https://github.com/caskroom/homebrew-cask) takes binaries, if that's what you were thinking of.